plugin/evm/imports_test.go

```diff
@@ -54,10 +56,11 @@ func TestMustNotImport(t *testing.T) {
 		// Importing these packages configures libevm globally and it is not
 		// possible to do so for both coreth and subnet-evm, where the client may
 		// wish to connect to multiple chains.
-		"plugin/evm/atomic": {"core", "core/vm", "params"},
-		"plugin/evm/client": {"core", "core/vm", "params"},
-		"plugin/evm/config": {"core", "core/vm", "params"},
+		"plugin/evm/atomic": {"plugin/evm/customtypes", "params"},
+		"plugin/evm/client": {"plugin/evm/customtypes", "params"},
 		"plugin/evm/header": {"core", "core/vm", "params"},
+		"ethclient":         {"plugin/evm/customtypes", "params"},
+		"warp":              {"plugin/evm/customtypes", "params"},
```

Comment:
Should we still also be checking for "core" and "core/vm" not being imported by these?

Reply:
I don't think the intention was ever to check `core` and `core/vm` but rather check the libevm registration in `params` and `customtypes` (indirectly)

Reply:
Others would know better than me, but this comment seems to suggest the intention was to explicitly check that `core` and `core/vm` are not imported by these packages, right?
```
        // The following sub-packages of plugin/evm must not import core, core/vm
		// so clients (e.g., wallets, e2e tests) can import them without pulling in
		// the entire VM logic.
```

Reply:
Libevm registration happens in the following packages:

- `params`
- `plugin/evm/customtypes`
- `core`

and `core/vm` no longer exists.

So the list of packages to avoid importing for libevm double registration should be `{"core", "plugin/evm/customtypes", "params"}`.

Reply:
Regarding the removal of `plugin/evm/config`, it was originally added in https://github.com/ava-labs/coreth/pull/734 - maybe we should keep it? As long as it doesn't bother us as well 🤷

Reply:
I'm pretty sure `params` pkg was the reason, but I don't want to block this any further and reverting any extra changes.

Reply:
and also by transition `core` cannot be ever imported because we already marked `customtypes` and `params`(which are already imported in `core`) as forbidden. but as I said I don't want to drag this conversation anymore.

Reply:
Sounds like I was just misunderstanding the tests intent based on the comment or missing context potentially, sorry. But sounds good